SetupCrossTenantRelationshipForTargetTenant.ps1

```diff
@@ -729,7 +772,7 @@ function Run-ExchangeSetupForTargetTenant([string]$targetTenant, [string]$resour
                                         -ExchangeRemoteMove:$true
     }
 
-    if ($Error[0].Exception.ErrorRecord.FullyQualifiedErrorId.Contains("MaximumConcurrentMigrationLimitExceededException"))
+    if ($Error[0] -eq $false -and $Error[0].Exception.ErrorRecord.FullyQualifiedErrorId.Contains("MaximumConcurrentMigrationLimitExceededException"))

```

Comment:
❓
```ps
PS C:\> $error[0]
PS C:\> $error[0] -eq $false
False
PS C:\> asdfasdf
asdfasdf : The term 'asdfasdf' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the
name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ asdfasdf
+ ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (asdfasdf:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

PS C:\> $error[0]
asdfasdf : The term 'asdfasdf' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the
name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ asdfasdf
+ ~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (asdfasdf:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

PS C:\> $error[0] -eq $false
False
```

Reply:
The problem here is we were hitting a null object error because this was empty:

$Error[0].Exception.ErrorRecord.FullyQualifiedErrorId.Contains("MaximumConcurrentMigrationLimitExceededException"

Here I am adding an additional check that is saying $error[0] is populated and then performing the check against that error.

In my testing I was able to consistently reproduce the null object until making this change, after the change there were no errors for null object.

Is my logic wrong here?

Reply:
not sure, just wanted to say `$error[0] -eq $false` returns the same value regardless of how `$error` is being configured

Reply:
(my powershell version):

```
PS C:\> (Get-Host).Version

Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      19041  1023
```